FHEM/10_SD_GT.pm

```diff
@@ -61,407 +61,410 @@ sub checkVersion;
-											'1' => 'off',
-											'2' => 'off',
-											'9' => 'off',
-											'A' => 'off',
+            'C' => { # unit C
+                      'unit' => 'C',
+                      '3' => 'on',
+                      '4' => 'on',
+                      '7' => 'on',
+                      'B' => 'on',
+                      '1' => 'off',
+                      '2' => 'off',
+                      '9' => 'off',
+                      'A' => 'off',
                     },
-						'2' => { # unit 2
-											'unit' => 'D',
-											'1' => 'on',
-											'2' => 'on',
-											'9' => 'on',
-											'A' => 'on',
-											'3' => 'off',
-											'4' => 'off',
-											'7' => 'off',
-											'B' => 'off',
+            '2' => { # unit 2
+                      'unit' => 'D',
+                      '1' => 'on',
+                      '2' => 'on',
+                      '9' => 'on',
+                      'A' => 'on',
+                      '3' => 'off',
+                      '4' => 'off',
+                      '7' => 'off',
+                      'B' => 'off',
                     },
-						'A' => { # unit A
-											'unit' => 'all',
-											'1' => 'on',
-											'2' => 'on',
-											'9' => 'on',
-											'A' => 'on',
-											'3' => 'off',
-											'4' => 'off',
-											'7' => 'off',
-											'B' => 'off',
+            'A' => { # unit A
+                      'unit' => 'all',
+                      '1' => 'on',
+                      '2' => 'on',
+                      '9' => 'on',
+                      'A' => 'on',
+                      '3' => 'off',
+                      '4' => 'off',
+                      '7' => 'off',
+                      'B' => 'off',
                     },
-					}
+          }
 );
 
 sub Define {
-	my ($hash, $def) = @_;
-	my @a = split m{\s+}xms , $def;
-	my $name = $hash->{NAME};
-	my $iodevice;
-	my $ioname;
-
-	if( @a < 3 ) { return 'SD_GT: wrong syntax for define, must be: define <name> SD_GT <DEF> <IODev>' };
-
-	$hash->{DEF} = $a[2];
-	if ($a[3]) { $iodevice = $a[3] }
-	if ($a[4]) { readingsSingleUpdate($hash,'SystemCode',$a[4],1) }
-	if ($a[5]) { readingsSingleUpdate($hash,'Version',$a[5],1) }
-	$hash->{VersionModule} = $VERSION;
-
-	$modules{SD_GT}{defptr}{$hash->{DEF}} = $hash;
-	if (exists $modules{SD_GT}{defptr}{ioname} && !$iodevice) { $ioname = $modules{SD_GT}{defptr}{ioname} };
-	if (not $iodevice) { $iodevice = $ioname }
-
-	AssignIoPort($hash, $iodevice);
-	return;
+  my ($hash, $def) = @_;
+  my @a = split m{\s+}xms , $def;
+  my $name = $hash->{NAME};
+  my $iodevice;
+  my $ioname;
+
+  if( @a < 3 ) { return 'SD_GT: wrong syntax for define, must be: define <name> SD_GT <DEF> <IODev>' };
+
+  $hash->{DEF} = $a[2];
+  if ($a[3]) { $iodevice = $a[3] }
+  if ($a[4]) { readingsSingleUpdate($hash,'SystemCode',$a[4],1) }
+  if ($a[5]) { readingsSingleUpdate($hash,'Version',$a[5],1) }
+  $hash->{VersionModule} = $VERSION_SD_GT;
```

Comment:
dito

Reply:
Möchtest du nun dein eigenes Modul überdenken ;-)
Ich denke mal, das Dito bezieht sich auf die Zeile 195 und nicht alle grün markierten Zeilen.

Reply:
Das bezieht sich natürlich auf die Zeile 195. Github zitiert halt immer einige Zeilen mehr und da du Tabs in Leerzeichen gewandelt hast, sieht man den Wald vor lauter Bäumen nicht mehr :-)